### PR TITLE
add test for empty method

### PR DESCRIPTION
**Pull request by renman-ymd**
Close #17

### CODE REVIEW

File: src/Insight/IGTest.class.st [hunk 1]
```diff
@@ -223,6 +227,23 @@ IGTest >> testCoverageInstrumentAStatementWithAnAfterHook [
...
+	
+	counter := IGCounter new.
+	afterHook := IGAfterStatementHook new do: counter.
+
+	instrumentation := IGInstrumentation new addHook: afterHook.
+	methodToInstrument := IGTest >> #emptyMethod.
+	
+	instrumentation instrumentMethod: methodToInstrument during: [ self emptyMethod ].
+	
+	self assert: counter count equals: 0
```
Comment (FedeLoch) [inline src/Insight/IGTest.class.st:243]: This brings the question of "how can we check that the method was executed (even if it is an empty method) using Insight?"
